src/content/blog/astro-420.mdx

```diff
@@ -0,0 +1,153 @@
+---
+title: "Astro 4.2"
+description: "Astro 4.2 is out now! This release includes two new experimental features to try out, improvements to accessibility rules, and more."
+publishDate: "January 18, 2024"
+authors:
+  - erika
+  - ema
+  - matthew
+  - nate
+  - bjorn
+coverImage: "/src/content/blog/_images/astro-420/post-header-4.2.webp"
+socialImage: "/src/content/blog/_images/astro-420/og-image-4.2.webp"
+lang: "en"
+---
+
+Astro 4.2 is now available! This release includes new experimental features to try out, improvements to accessibility rules, the ability for remark plugins to customize image optimization in Markdown, and many more improvements and bug fixes.
+
+Highlights include:
+
+- **[(Experimental) Support for prerendering pages using the Speculation Rules API](#experimental-prerendering-pages-using-the-speculation-rules-api)**
+- **[(Experimental) Reworked routing priority for injected routes](#experimental-reworked-routing-priority-for-injected-routes)**
+- **[New `redirectToDefaultLocale` config option](#new-redirecttodefaultlocale-config-option)**
+- **[Improved accessibility rules](#improved-accessibility-rules)**
+- **[Customizable image optimization in Markdown](#customizable-image-optimization-in-markdown)**
+
+As a side note, this is the first release of Astro to majorly consist of outside contributions, with all but one highlights of this release being from contributors. This is a huge milestone for us, as it means that Astro is now big enough to have a thriving community of contributors capable of spearheading releases. Thank you to everyone who contributed to this release!
```

Comment:
```suggestion
This marks the first Astro release of almost all community-built features, with only one highlight brought to you by an Astro core maintainer. This is a huge milestone for us, as it means that Astro is now big enough to have a thriving community of contributors capable of spearheading releases. Thank you to everyone who contributed to this release!
```
There were an awful lot of "c" words between community and contributor/ing, consisting, coming from... so I tried to vary it a bit!  :smile: : 

I also replaced "outside" because that can feel a bit like "these people are outsiders" vs they just don't happen to be core. They are "inside" our community!

Same idea re: "side note"... maybe this is a Big Deal instead of a side note!

Reply:
This looks great to me! I couldn't find a clever way to write it, but inverting the statement like you did works awesomely